core/src/main/java/com/hevelian/identity/core/model/Tenant.java

```diff
@@ -48,6 +45,12 @@
   @Column(nullable = false)
   private String contactEmail;
 
+// Not all JDBC drivers support Blob locators.
+  @Lob
+  @Column(length=100000)
```

Comment:
Where did you take this number from?

Reply:
It's my mistake) This colum should be without limiter.

Reply:
Leave it here. this is a great check to forbid storing here huge files. In other case application will crash with OutOfMemory. Checking size is not just a validation. This is related to me previous reply.

Reply:
Just set the limit to some value which is equals to certain amount of KB.